### PR TITLE
removing remaining references to JSON

### PR DESCRIPTION
**Pull request by Sakurann**
Clarifying that "There is no such thing as `vc+json`", addressing #1044 and preventing future confusion like in #1088 after it was merged. As part of that clarification, it references JSON-LD spec directly wrt the processing of `@context`, since if the base media type is `vc+ld+json`, it needs to be a valid JSON-LD.

***
<a href="https://pr-preview.s3.amazonaws.com/Sakurann/vc-data-model/pull/1182.html" title="Last updated on Jul 1, 2023, 9:49 PM UTC (8278dc6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1182/562b9f3...Sakurann:8278dc6.html" title="Last updated on Jul 1, 2023, 9:49 PM UTC (8278dc6)">Diff</a>

### CODE REVIEW

File: index.html
Comment (dlongley) [inline index.html:1128]: Well, we should avoid possibly deviating from the JSON-LD spec itself:

> JSON-LD is used in many specifications as the specified data format. However, there is also a desire to allow some JSON-LD contents to be processed as plain JSON, without using any of the JSON-LD algorithms. Because JSON-LD is very flexible, some terms from the original format may be locally overridden through the use of [embedded contexts](https://www.w3.org/TR/json-ld11/#dfn-embedded-context), and take a different meaning for JSON-LD based implementations. On the other hand, "plain JSON" implementations may not be able to interpret these [embedded contexts](https://www.w3.org/TR/json-ld11/#dfn-embedded-context), and hence will still interpret those terms with their original meaning. To prevent this divergence of interpretation, JSON-LD 1.1 allows term definitions to be protected.

https://www.w3.org/TR/json-ld11/#protected-term-definitions

We can instead just reference JSON-LD directly to make it very clear that that spec is what is to be used to understand `@context`. Perhaps like this:

```suggestion
This specification requires for a <code>@context</code> <a>property</a>
to be present; this property is defined by [[JSON-LD]].
```

**Review by dlongley — CHANGES_REQUESTED** [review]

File: index.html
Comment (Sakurann) [reply index.html:1128]: I really like the suggestion! Thanks.

File: index.html
Comment (msporny) [inline index.html:615]:
```suggestion
a valid document.
```

**Review by msporny — APPROVED** [review]
I'm a bit on the fence about removing this language, as that is how some implementations (including some of Digital Bazaar's processing pipelines), handle the VC as it is coming into our systems. We do, when it matters, do full JSON-LD processing, but that is not always necessary to perform. As long as we don't suggest that full JSON-LD processing is always required, this PR is fine. We might want to move a variation of this text to the implementation guide to ensure that developers know that it's ok to 1) ensure the order of the `@context` entries, and then 2) process as ordinary JSON, in some circumstances.

**Review by dlongley — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by decentralgabe — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by OR13** [comment]
Keep in mind that if `+ld+json` is not registered... none of what we are doing will matter... because there will be no general guidance on processing the media types we are requesting registration for.

https://github.com/w3c/json-ld-syntax/pull/415

**Review by OR13 — CHANGES_REQUESTED** [review]
I'm requesting changes until the structured suffixes details are resolved sufficiently to protect implementers from the confusion that would arise from removing these references while not defining how the suffixes are processed.

**Comment by Sakurann** [comment]
@OR13 what concrete details on the structures suffixes do you want to be resolved? multiple suffixes draft in IETF and how `+ld+jwt` is being processed is orthogonal to the changes in this PR and should block it.

**Comment by OR13** [comment]
@Sakurann I guess you are right, the problem will be... if `+ld+json` is never registered, and its the only media type suffix we rely on, then our document should not advance.

I am approving, with the hope that does not happen.

**Review by OR13 — APPROVED** [review]

**Comment by msporny** [comment]
@Sakurann, waiting on a typo fix before merging: https://github.com/w3c/vc-data-model/pull/1182/files#r1249374162

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-07-11-vcwg-special#section1-7) on 2023-07-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.7. removing remaining references to JSON (pr vc-data-model#1182)

_See github pull request [vc-data-model#1182](https://github.com/w3c/vc-data-model/pull/1182)._




**Brent Zundel:** this P.R removes differentiation between JSONLD and JSON. Has relatively broad approval.  
… minor changes from manu, that I agree with.  
… not sure this P.R calls for much discussion, but open to any if there are some.  

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-07-11-vcwg-special#section1-7) on 2023-07-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.7. removing remaining references to JSON (pr vc-data-model#1182)

_See github pull request [vc-data-model#1182](https://github.com/w3c/vc-data-model/pull/1182)._




**Brent Zundel:** this P.R removes differentiation between JSONLD and JSON. Has relatively broad approval.  
… minor changes from manu, that I agree with.  
… not sure this P.R calls for much discussion, but open to any if there are some.  

</details>

**Comment by msporny** [comment]
> @Sakurann, waiting on a typo fix before merging: https://github.com/w3c/vc-data-model/pull/1182/files#r1249374162

Actually, the suggestion is Editorial, and made by me, so I can just merge at this point and make the change, which is what I'll do.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-07-11-vcwg-special#section1-7) on 2023-07-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.7. removing remaining references to JSON (pr vc-data-model#1182)

_See github pull request [vc-data-model#1182](https://github.com/w3c/vc-data-model/pull/1182)._




**Brent Zundel:** this P.R removes differentiation between JSONLD and JSON. Has relatively broad approval.  
… minor changes from manu, that I agree with.  
… not sure this P.R calls for much discussion, but open to any if there are some.  

</details>

**Pull request merged**
